common/src/proto.rs

```diff
@@ -110,6 +110,9 @@ pub struct Character {
     pub state: CharacterState,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct InactiveCharacter(pub Character);
```

Comment:
That might be a good idea. Currently, only `server`'s `sim.rs` uses this struct, so making it a private struct in `sim` might be the right move. Just moving it to the server crate won't guarantee that the client code won't see it, since `client` also has to depend on `server` because of single-player.